### ISSUE TITLE
Sending a Complex List corrupts memory

### ISSUE THREAD

**Issue by adriweb** [CE-Programming/CEmu#80]
This one, for instance: [ComplexList.8xl.zip](https://github.com/CE-Programming/CEmu/files/491690/ComplexList.8xl.zip) (it's `{1+i, -3+2i, 4}`)
The VAT is probably messed up after transferring it, as the Mem menu shows bad tokens, and the Lists menu takes ages to load (speed it up in unthrottled) and when it does, the entries are bad.

As far as I could test, *every* other vartype (well, except app and OS) is working fine.

**Comment by adriweb** [comment]
Initial investigation about that list:

```
data_size       73
data_size_high  0
data_size_low   73
data_start      55
var_offset      55
var_ptr         64
var_size        0
var_size_high   0
var_size_low    56
var_type        13
```

**Comment by adriweb** [comment]
Fixed in 735d216
